api/server/router/build/build_routes.go

```diff
@@ -216,18 +216,18 @@ func (br *buildRouter) postPrune(ctx context.Context, w http.ResponseWriter, r *
 			opts.MaxUsedSpace = bs
 		}
 
-		if bs, err := parseBytesFromFormValue("min-free-space"); err != nil {
+		bs, err := parseBytesFromFormValue("min-free-space")
+		if err != nil {
 			return err
-		} else {
-			opts.MinFreeSpace = bs
 		}
+		opts.MinFreeSpace = bs
```

Comment:
Could use some input from others on these; before this, the variables would be explicitly scoped to the `if/else`; after this they get a wider scope. Not a _huge_ issue (as they're still close together), but wondering what we prefer for these.